src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int INIT_RADIUS = 10;
+    private static final String INIT_COLOR = "white";
+    private static final int INIT_FIGURES = 5;
+    private static final int MAX_SIZE = 100;
+
+    public Figure getRandomFigure() {
+
+        switch (new Random().nextInt(INIT_FIGURES)) {
+            case 0:
+                return new Circle(
+                        new ColorSupplier().getRandomColor(), 
+                        new Random().nextInt(MAX_SIZE));
+            case 1:
+                return new IsoscelesTrapezoid(
+                        new ColorSupplier().getRandomColor(), 
+                        new Random().nextInt(MAX_SIZE),
+                        new Random().nextInt(MAX_SIZE),
+                        new Random().nextInt(MAX_SIZE));
+            case 2:
+                return new RightTriangle(
+                        new ColorSupplier().getRandomColor(),
+                        new Random().nextInt(MAX_SIZE),
+                        new Random().nextInt(MAX_SIZE));
+            case 3:
+                return new Rectangle(
+                        new ColorSupplier().getRandomColor(),
+                        new Random().nextInt(MAX_SIZE),
+                        new Random().nextInt(MAX_SIZE));
+            case 4:
+                return new Square(
+                        new ColorSupplier().getRandomColor(),
+                        new Random().nextInt(MAX_SIZE));
```

Comment:
create a separate method for each of figures that will be returning random figure, for example, getRandomRightTriangle() and so on